### PR TITLE
Float-friendly replacement for Django's built-in `add` filter.

### PR DESCRIPTION
**Pull request by centricle**
Float-friendly replacement for Django's built-in `add` filter.

### CODE REVIEW

**Comment by dbrgn** [comment]
Thanks! Calling it `sum` solves the `add` conflict :)

File: mathfilters/templatetags/mathfilters.py [hunk 1]
```diff
@@ -119,3 +119,18 @@ def mod(value, arg):
         except Exception:
             return ''
 mod.is_safe = False
+
+
+@register.filter(name='sum')
+def sum(value, arg):
```
Comment (dbrgn) [inline mathfilters/templatetags/mathfilters.py]: Maybe call the function `addition` to avoid overloading Python's `sum` builtin?

**Comment by dbrgn** [comment]
Also, maybe add 2-3 tests involving `Decimal` (see other tests) to avoid decreasing the test coverage?

**Comment by centricle** [comment]
Huh. Coveralls is complaining about existing tests that weren't touched. I wonder if something changed on their end.

**Comment by dbrgn** [comment]
Huh, strange. Anyways, your PR looks good. Thanks!

**Pull request merged**